src/renderer/src/routes/pages/BuyAndSellTokens/BuyTokensAccountItem.tsx

```diff
@@ -0,0 +1,56 @@
+import { useTranslation } from 'react-i18next'
+import { MdOutlineContentCopy } from 'react-icons/md'
+import { BlockchainIcon } from '@renderer/components/BlockchainIcon'
+import { IconButton } from '@renderer/components/IconButton'
+import { Loader } from '@renderer/components/Loader'
+import { Tooltip } from '@renderer/components/Tooltip'
+import { NumberHelper } from '@renderer/helpers/NumberHelper'
+import { StringHelper } from '@renderer/helpers/StringHelper'
+import { UtilsHelper } from '@renderer/helpers/UtilsHelper'
+import { useBalance } from '@renderer/hooks/useBalances'
+import { useCurrencySelector } from '@renderer/hooks/useSettingsSelector'
+import { IAccountState } from '@shared/@types/store'
+
+type TProps = {
+  account: IAccountState
+}
+
+export const BuyTokensAccountItem = ({ account }: TProps) => {
+  const { t } = useTranslation('pages', { keyPrefix: 'buyAndSellTokens.buyTokensAccountsSide' })
+  const { currency } = useCurrencySelector()
+  const { data, isLoading } = useBalance(account)
+  const { address } = account
```

Comment:
Separate these lines by a line break. It easier to understand and separate what is hooks or simple const